packages/module/release.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  branches: [{ name: 'main', channel: 'prerelease' }],
+  branches: [{ name: 'main', channel: 'alpha' }],
```

Comment:
I think if you don't have a prerelease channel semantic-release will throw errors.

Also tagFormat will need to be updated. [Catalog view release config for reference.](https://github.com/patternfly/react-catalog-view/blob/main/packages/module/release.config.js).

You also might want to enable dryRun temporarily to make sure everything works right before we start actually putting releases out.

Reply:
@wise-king-sullyman All set thanks.